src/state/session/index.tsx

```diff
@@ -373,9 +374,14 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
     [setState, initSession],
   )
 
+  /**
+   * Clears the `currentAccount` from session. Typically used to drop the user
+   * back to the sign-in page.
+   */
   const clearCurrentAccount = React.useCallback(() => {
     setStateAndPersist(s => ({
       ...s,
+      agent: PUBLIC_BSKY_AGENT,
```

Comment:
This is a soft log out. The user was authed and wanted to sign in with a different account.